### ISSUE TITLE
Can't complete `:tag` with `tagfunc` set to `v:lua.vim.lsp.tagfunc`

### ISSUE THREAD

**Issue by d12bb** [hrsh7th/cmp-cmdline#51]
When using the LSP for tagfunc (see `:h vim.lsp.tagfunc()`), cmp complains that there's no tags file when using the `:tag` command. As this command is fully functioning without a tags file, this should not throw errors and instead complete just fine.

<img width="1277" alt="image" src="https://user-images.githubusercontent.com/16717303/168311065-1c081b16-631c-4681-9bb5-312f9a56f371.png">

**Comment by Clejd** [comment]
I have the same issue. That happens with ts (tree sitter) command too

**Comment by weilbith** [comment]
Yes. It seems like no tagfunc works and the plugin goes straight for the tagfile. 
Does anyone know if it is possible to ignore a list of command or a regular expression to disable completion for it?
